rnote-ui/data/ui/workspacebrowser.ui

```diff
@@ -41,13 +41,35 @@
                       <object class="GtkListBoxRow" id="dir_up_row">
                         <property name="tooltip_text" translatable="yes">Move directory up</property>
                         <child>
-                          <object class="GtkLabel">
-                            <property name="margin-top">3</property>
-                            <property name="margin-bottom">3</property>
-                            <property name="margin-start">12</property>
-                            <property name="margin-end">12</property>
-                            <property name="halign">start</property>
-                            <property name="label">‥</property>
+                          <object class="GtkBox" id="workspace_button_box">
```

Comment:
This box should be the parent of the `files_prefix_listbox`. And the box containing the button should be moved so that it and the listbox are siblings to each other.

Reply:
I'm a bit unsure what you mean, could you please draw a picture?

Reply:
do you mind that I commit to the PR with that change? I think it is easier if I just do it, rather than try to explain it

Reply:
I fixed it. Doing UI is just a lot of trial-and-error and trying out different things, I think it would have been cumbersome trying to explain in a lengthy discussion